manifests/cf-manifest/outputs/cf-ssl-certificates.yml

```diff
@@ -1,125 +1,116 @@
 secrets:
-  router_ssl_cert:  |
+  bbs_ca_cert: |
```

Comment:
This first commit is quite difficult to read because it appears to have introduced new items and re-ordered existing items at the same time. Could you split it into two separate commits?

It would also be good to add the `certstrap` commands that you used to the commit message for future reference.